contracts/NiftyFootballTradingCardBlindPack.sol

```diff
@@ -127,7 +131,7 @@ contract NiftyFootballTradingCardBlindPack is Ownable, Pausable {
         } else {
             // any trapped ether can be withdrawn with withdraw()
             totalPurchasesInWei = totalPurchasesInWei.add(msg.value);
-            wallet.transfer(msg.value);
+            splitFunds(totalPrice(_numberOfCards));
```

Comment:
can we save the duplicate call to `totalPrice(_numberOfCards)`?

Reply:
done